src/main/java/com/passpnu/passwordmanager/Dto/ServiceDto.java

```diff
@@ -0,0 +1,4 @@
+package com.passpnu.passwordmanager.Dto;
```

Comment:
Refer to https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html for Java-compliant package naming

Reply:
fixed